src/grandorgue/model/GODrawStop.cpp

```diff
@@ -39,6 +39,45 @@ void GODrawstop::RegisterControlled(GODrawstop *sw) {
   m_ControlledDrawstops.push_back(sw);
 }
 
+void GODrawstop::UnRegisterControlled(GODrawstop *sw) {
```

Comment:
This is not only about tests, correct?

Reply:
It is used for tests

Reply:
I usually don't recommend to add code in core to be used exclusively by tests. That's a weird design.

Reply:
But initially the core was designed to be set up only from the ODF file, not from the testing code. So I had to add some set up/clean up code for using in tests.

I think that is a right pattern: to replace the Load() code with calls to certain (new) setting up methods and to use the same setting up methods in the tests.

Reply:
Ok, but I think it's because of wrong implementation of MVC as they should be independent and so instantiated separately.

Reply:
@oleg68 adding comments to these should be great!